desktop/src/renderer/src/components/transfer/lan-views/LanSubmissionAgreementView.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+  Button,
+  Radio,
+  RadioGroup,
+} from "@bcgov/design-system-react-components";
+import { Box, Stack, Typography } from "@mui/material";
+import { SubAgreementScrollBox } from "@renderer/components";
+import { useState } from "react";
+import { DeclineSubAgreementModal } from "../DeclineSubAgreementModal";
+import { useNavigate } from "react-router-dom";
+
+type Props = {
+  accession: string;
+  application: string;
+  onNextPress: () => void;
+  onBackPress: () => void;
+};
+
+export const LanSubmissionAgreementView = ({
+  accession,
+  application,
+  onNextPress,
+  onBackPress,
+}: Props) => {
+  const navigate = useNavigate();
+  const [accept, setAccept] = useState<boolean | null>(null);
+  const [showDeclineModal, setShowDeclineModal] = useState<boolean>(false);
+
+  const AgreementBox = () => {
+    return (
+      <Stack
+        spacing={1}
+        sx={{
+          padding: "8px 16px",
+          backgroundColor: "var(--acc-app-confirmation-bg)",
+        }}
+      >
+        <Typography
+          sx={{
+            color: "var(--text)",
+            fontSize: "0.8em",
+          }}
+        >
+          <b>Do you accept the Submission Agreement?</b> *
```

Comment:
trivial issue - can we keep it consistent on the expression of "required", i.e
`*` or `(required)` or `(required)*`